lib/src/config.rs

```diff
@@ -72,7 +72,10 @@ impl FastlyConfig {
     /// Parse a `fastly.toml` file into a `FastlyConfig`.
     pub fn from_file(path: impl AsRef<Path>) -> Result<Self, FastlyConfigError> {
         fs::read_to_string(path.as_ref())
-            .map_err(Into::into)
+            .map_err(|err| FastlyConfigError::IoError {
+                path: format!("{}", path.as_ref().display()),
```

Comment:
```suggestion
                path: path.as_ref().to_string(),
```

types that implement `Display` automatically implement [`std::string::ToString`](https://doc.rust-lang.org/std/string/trait.ToString.html), so we can use that to get our owned string rather than a `format("{}", ...)` call :slightly_smiling_face:

Reply:
This doesn't work but `path.as_ref().display().to_string()` does.  Is that what you meant?

Reply:
ah! yes, thank you. i'm mostly concerned about avoiding `format!("{}"` here :slightly_smiling_face:

Reply:
Updated

Reply:
woohoo!